tests/Examples/tfhe_rust_bool/cpu/BUILD

```diff
@@ -1,21 +1,40 @@
-# See README.md for setup required to run these tests
-
-load("//bazel:lit.bzl", "glob_lit_tests")
+load("@heir//tests/Examples/tfhe_rust:test.bzl", "tfhe_rs_end_to_end_test")
+load("@rules_rust//rust:defs.bzl", "rust_binary")
 
 package(default_applicable_licenses = ["@heir//:license"])
 
-glob_lit_tests(
-    name = "all_tests",
-    data = [
-        "Cargo.toml",
-        "src/main.rs",
-        "src/main_bool_add.rs",
-        "@heir//tests:test_utilities",
+tfhe_rs_end_to_end_test(
+    name = "and_test",
+    size = "large",
+    heir_translate_flags = ["--emit-tfhe-rust-bool"],
+    mlir_src = "test_and.mlir",
+    test_src = "src/main.rs",
+)
+
+rust_binary(
```

Comment:
I have no problem keeping the rust binaries, but are they used? Maybe just omit them and remove the associated main code, and have these be test-only?

Reply:
Good point. All but this one is omitted, and I thought I should keep one rust_binary in to show an example of how to compile it for a binary, but on second thought, it's pretty simple to do so I'll remove it. This should probably be in a tutorial or something.

Reply:
Removed